uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/style/_select2.scss

```diff
@@ -1,4 +1,10 @@
 // removes extra padding on select2 ul element added by bootstrap:
-.select2-container ul.select2-choices {
+.select2-container.crm-select2 ul.select2-choices {
   padding-left: 0;
 }
+
+// This fixes the arrow dropdown width by removing 4 padding and and adding 8 width:
+.select2-container.crm-select2 .select2-choice .select2-arrow {
+  padding: 0;
```

Comment:
Do you need to flush all padding or may you just flush just padding-right?

Reply:
All paddings need to be removed. If I remove padding-right it will look as if it were aligned to the right.